canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -235,6 +236,28 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 				</div>
 			</div>
 		);
+		const tooltipId = uuid4() + "-tooltip-expr-table-cell";
```

Comment:
Do we have this same code in other places?  I'm wondering if we shouldn't create a new component that handles this.  It would simply the code slightly and we could reuse it in other tables.

Reply:
Yes we have this code in multiple places. I will create a reusable component.